### PR TITLE
Reader: Add a8c conversations

### PR DESCRIPTION
**Pull request by blowery**
Adds a "A8C Conversations" sidebar item and stream. No pics.

You'll likely need to restart calypso after switching to this branch to pick up the configuration changes.

### CODE REVIEW

File: client/reader/sidebar/index.jsx [hunk 1]
```diff
@@ -144,8 +145,34 @@ export const ReaderSidebar = createReactClass( {
...
+										'/read/conversations/a8c',
+										this.props.path,
+										{
+											'sidebar-streams__conversations': true,
+										},
+									) }
+								>
+									<a href="/read/conversations/a8c">
+										<svg
+											className={ 'gridicon gridicon-automattic' }
```
Comment (samouri) [inline client/reader/sidebar/index.jsx]: don't we have gridicons for this now?

Comment (blowery) [inline client/reader/sidebar/index.jsx]: no, the a8c logo isn't in gridicons last i checked....

File: client/reader/sidebar/index.jsx [hunk 2]
```diff
@@ -144,8 +145,34 @@ export const ReaderSidebar = createReactClass( {
...
+											width="24"
+											height="24"
+											xmlns="http://www.w3.org/2000/svg"
+											viewBox="0 0 16 16"
+										>
+											<path d="M7.99 1.57C3.75 1.57 1 4.57 1 7.8v0.4c0 3.18 2.75 6.24 6.99 6.24 4.26 0 7.01-3.05 7.01-6.24V7.8C15 4.57 12.25 1.57 7.99 1.57zM12.74 8.13c0 2.32-1.69 4.42-4.74 4.42 -3.05 0-4.73-2.1-4.73-4.42V7.84c0-2.32 1.67-4.38 4.73-4.38 3.06 0 4.75 2.07 4.75 4.39V8.13z" />
+											<path d="M9.47 5.73C9.07 5.47 8.52 5.59 8.26 6L6.21 9.17c-0.26 0.41-0.15 0.95 0.26 1.21 0.4 0.26 0.95 0.14 1.21-0.26l2.05-3.17C9.99 6.53 9.88 5.99 9.47 5.73z" />
+										</svg>
+										<span className="menu-link-text">
+											{ this.props.translate( 'A8C Conversations' ) }
```
Comment (a8ci18n) [inline client/reader/sidebar/index.jsx]: Hi! I've found a possible matching string that has already been translated **21** times:
`translate( 'Conversations' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Comment (yoavf) [inline client/reader/sidebar/index.jsx]: Let's not translate this, it's A8C only

Comment (blowery) [inline client/reader/sidebar/index.jsx]: done!

**Comment by blowery** [comment]
tests are failing here because of the sidebar... just live with it or add the ignores?

**Comment by blowery** [comment]
> tests are failing here because of the sidebar... just live with it or add the ignores?

ignoring them

File: client/reader/conversations/controller.js [hunk 3]
```diff
@@ -19,7 +19,7 @@ export function conversations( context ) {
 	const title = 'Reader > Conversations';
 
 	trackPageLoad( basePath, 'Reader > Conversations', mcKey );
-	recordTrack( 'calypso_reader_discover_viewed' );
+	recordTrack( 'calypso_reader_conversations_viewed' );
```
Comment (samouri) [inline client/reader/conversations/controller.js]: snuck in this fix :)

**Pull request merged**